### PR TITLE
(DOCSP-14948): [CLI V2] Enforce a Document Schema

### PR DESCRIPTION
**Pull request by nlarew**
### Issue JIRA link:

- (DOCSP-14948): [CLI V2] Enforce a Document Schema

### Docs staging link (requires sign-in on MongoDB Corp SSO):

- [MongoDB > Enforce a Document Schema (Realm CLI)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/cli/mongodb/enforce-a-document-schema)

### Review Guidelines
[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Comment by nlarew** [comment]
cc @mongomoe @MongoCaleb
LFR + we can use this as an example for other procedure updates.

Note:
- `import` -> `push`, `export` -> `pull`
- `/includes/note-procedure-uses-cli-v1.rst` -> `/includes/note-procedure-uses-cli-v2.rst`

File: source/includes/steps-enforce-a-document-schema-import-export.yaml [hunk 1]
```diff
@@ -1,68 +1,56 @@
...
+     
+     If you haven't already defined rules or a schema for the collection, you
+     need to manually create its configuration directory and ``schema.json``:
+
+     .. code-block:: bash
+        
+        # Create the collection's configuration directory
+        mkdir -p data_sources/<service>/<db>/<collection>
+        # Create the collection's schema file
+        touch data_sources/<service>/<db>/<collection>/schema.json
```
Comment (MongoCaleb) [inline source/includes/steps-enforce-a-document-schema-import-export.yaml:45]: I don't _think_ touch is a supported command in Windoze. Do we care?

**Review by MongoCaleb — APPROVED** [review]
LGTMeow

File: source/includes/steps-enforce-a-document-schema-import-export.yaml [hunk 2]
```diff
@@ -1,68 +1,56 @@
...
+  
+  The root-level schema for every document must be be of type ``object``. You
```
Comment (mongomoe) [inline source/includes/steps-enforce-a-document-schema-import-export.yaml:47]:
```suggestion
  The root-level schema for every document must be of type ``object``. You
```

needed change ^

File: source/includes/steps-enforce-a-document-schema-import-export.yaml [hunk 3]
```diff
@@ -1,68 +1,56 @@
...
-
-     This guide focuses on creating a ``schema`` for the collection.
-     Check out the other configuration guides to learn how to
-     :doc:`define roles and permissions
-     </mongodb/define-roles-and-permissions>` and :doc:`create filters
-     </mongodb/filter-incoming-queries>`.
-
-  .. include:: /includes/data-lake-rules-note.rst
-
+  To define a document schema with {+cli-bin+} you need a local copy of your
```
Comment (mongomoe) [inline source/includes/steps-enforce-a-document-schema-import-export.yaml:15]:
```suggestion
  To define a document schema with {+cli-bin+}, you need a local copy of your
```
grammar nitpick ^

**Review by mongomoe — APPROVED** [review]
1 needed typo fix before lgtm

File: source/includes/steps-enforce-a-document-schema-import-export.yaml (hunk 1 above)
Comment (nlarew) [reply source/includes/steps-enforce-a-document-schema-import-export.yaml:45]: I can make it platform agnostic with `echo`!

**Pull request merged**